src/examples/libpmemobj/array/array.c

```diff
@@ -235,8 +243,9 @@ realloc_int(PMEMoid *info, size_t prev_size, size_t size)
 {
 	TOID(int) array;
 	TOID_ASSIGN(array, *info);
+
 	POBJ_REALLOC(pop, &array, int, size * sizeof(int));
-	for (int i = prev_size; i < size; i++)
+	for (int i = (int)prev_size; i < size; i++)
```

Comment:
this looks like a mindless compiler-issue fix. Please correct the underlying issue instead of silencing the warning.